ocs_ci/ocs/ui/views.py

```diff
@@ -1157,6 +1157,29 @@
 
 
 locators = {
+    "4.13": {
```

Comment:
@DanielOsypenko do we need to maintain a 4.13 locator in the release-4.12 branch?

Reply:
@ebenahar we have failures with mixed deployments OCP 4.13 & ODF 4.12. 
I assume it should also help with upgrade scenarios and post upgrade tests. I've put more info here -> https://github.com/red-hat-storage/ocs-ci/issues/7605

Reply:
ack. Thanks